mergin/client.py

```diff
@@ -469,6 +485,8 @@ def apply_push_changes(self, changes):
                 elif k == 'added':
                     shutil.copy(self.fpath(path), basefile)
                 elif k == 'updated':
+                    if "diff" not in item:
+                        continue
```

Comment:
Could you please add short comment why is this done as it is not very clear from the context...